src/open_clip/factory.py

```diff
@@ -61,6 +61,9 @@ def load_state_dict(checkpoint_path: str, map_location='cpu'):
 
 def load_checkpoint(model, checkpoint_path, strict=True):
     state_dict = load_state_dict(checkpoint_path)
+    # detect old format and make compatible with new format
+    if 'positional_embedding' in state_dict and not hasattr(model, 'positional_embedding'):
```

Comment:
Are we keeping this ? I thought we wanted to remove that conversion now

Reply:
@rom1504 I think it's still valid if someone wants to force a transition to the separate text tower model via config?

Reply:
ok